deps/config/apikey/sample.csv

```diff
@@ -4,4 +4,4 @@
 # sha256("admin-user")
 f470213d9ae659187a19b9cb2169b4b400544f4d3f59250eda657154700da616;tenant1:user1;*:*;admin
 # sha256("guest-user")
-8605f70ff5f55e2a9323d97de3dbf8e61f38314d93298ca00e19f8918fe8971b;tenant2:user2;*:secrets,*:keys,read:eth1accounts;signer
+8605f70ff5f55e2a9323d97de3dbf8e61f38314d93298ca00e19f8918fe8971b;tenant2:user2;*:secrets,*:keys,read:eth1accounts;
```

Comment:
Last column should be the role here, was it removed in code ?

Reply:
no role is needed for "guest-user" , this breaks the e2e tests.